### PR TITLE
Fix issue 4335

### LINKED ISSUE #4335: Admin.UnbanUser
We currently have org.couchers.admin.Admin.BanUser but if an error is made, there is no way to undo this action other than manually updating the database.

We also need org.couchers.admin.Admin.UnbanUser to reverse a ban in the unlikely event of an error.

### PR DESCRIPTION
**Pull request by lovesqualor**
I think this closes #4335? Not sure if there are more changes I need to make. 

**Backend checklist**
- [ ] Formatted my code by running `ruff check --select I --fix . && ruff check . && ruff format .` in `app/backend`
- [ ] Added tests for any new code or added a regression test if fixing a bug
- [ ] All tests pass
- [ ] Run the backend locally and it works
- [ ] Added migrations if there are any database changes, rebased onto `develop` if necessary for linear migration history

**Web frontend checklist**
- [ ] Formatted my code with `make format`
- [ ] There are no warnings from `make lint`
- [ ] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: app/web/.env.development [hunk 1]
```diff
@@ -1,9 +1,12 @@
-NEXT_PUBLIC_COUCHERS_ENV=preview
```
Comment (aapeliv) [inline app/web/.env.development]: You shouldn't commit these changes

File: app/backend/src/couchers/servicers/admin.py [hunk 2]
```diff
@@ -159,6 +159,14 @@ def BanUser(self, request, context):
...
             user.is_banned = True
         return self.AddAdminNote(request, context)
+    
+    def UnbanUser(self, request, context):
+        with session_scope() as session:
+            user = session.execute(select(User).where_username_or_email_or_id(request.user)).scalar_one_or_none()
+            if not user:
+                context.abort(grpc.StatusCode.NOT_FOUND, errors.USER_NOT_FOUND)
+            user.is_banned = False
+        return self.AddAdminNote(request, context)
```
Comment (aapeliv) [inline app/backend/src/couchers/servicers/admin.py:169]: The formatter is complaining, you should run
```
ruff check --select I --fix . && ruff check . && ruff format .
```

**Review by aapeliv — CHANGES_REQUESTED** [review]
Looking good, see the few small notes.

**Comment by aapeliv** [comment]
> I think this closes #1234?

That's the wrong issue it seems

**Comment by lovesqualor** [comment]
OK i fixed them

**Review by aapeliv — CHANGES_REQUESTED** [review]
Some tests are still failing. Please re-run the formatter.

**Pull request merged**